packages/experimental/Button/src/ToggleButton/ToggleButtonColorTokens.win32.ts

```diff
@@ -6,21 +6,15 @@ export const defaultToggleButtonColorTokens: TokenSettings<ToggleButtonTokens, T
   checked: {
     color: t.colors.neutralForeground1Selected,
     backgroundColor: t.colors.neutralBackground1Selected,
-    borderColor: t.colors.neutralStroke1Selected,
-    hovered: {
```

Comment:
Why are we removing the hovered definitions?

Reply:
Hover state for Toggle button is the same as for the default one.